docs/guides/integrating-the-feature-hub.md

```diff
@@ -22,7 +22,7 @@ multiple Feature Apps that share state through Feature Services:
    React][placing-feature-apps-on-a-web-page-using-react].
 
 To simplify the first three steps, an integrator can use the `createFeatureHub`
-function that is also provided by [`@feature-hub/core`][core-api]:
+function that is also provided via the [`@feature-hub/core`][core-api] package:
```

Comment:
I think `by` was more aligned with the wording above. The package is the actor here.